actors/runtime/exitcode/common.go

```diff
@@ -0,0 +1,18 @@
+package exitcode
+
+// Common error codes that may be shared by different actors.
+// Actors may also define their own codes, including redefining these values.
+
+const (
+	// Indicates a method parameter is invalid.
+	ErrIllegalArgument = FirstActorErrorCode + iota
+	// Indicates a requested resource does not exist.
+	ErrNotFound
+	// Indicates an action is disallowed.
+	ErrForbidden
+	// Indicates a balance of funds is insufficient.
+	ErrInsufficientFunds
+
+	// An error code intended to be replaced by different code structure or a more descriptive error.
+	ErrPlaceholder = ExitCode(1000)
```

Comment:
most ErrPlaceholder use sites are related state not found in `ipld.Get`, replaced with ErrNotFound?

Reply:
THey're all actor state loading, which I intend to clean up entirely.